gammapy/modeling/threeML_wrapper.py

```diff
@@ -0,0 +1,53 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""3ml minimization wrapper"""
+import numpy as np
+from astromodels import Parameter as AstroParameter
+from .iminuit import MinuitLikelihood
+
+
+def optimize_3ml(parameters, function, store_trace=False, **kwargs):
+    """iminuit optimization.
+
+    Parameters
+    ----------
+    parameters : `~gammapy.modeling.Parameters`
+        Parameters with starting values.
+    function : callable
+        Likelihood function.
+    store_trace : bool, optional
+        Store trace of the fit. Default is False.
+    **kwargs : dict
+        Options passed to `iminuit.Minuit` constructor. If there is an entry
+        'migrad_opts', those options will be passed to `iminuit.Minuit.migrad()`.
+
+    Returns
+    -------
+    result : (factors, info, optimizer)
+        Tuple containing the best fit factors, some information and the optimizer instance.
+    """
+    from threeML.minimizer.minimization import get_minimizer
```

Comment:
I am wondering whether one should not protect this import by raising a clear error message if threeML is not installed...